x-pack/plugins/observability_solution/entity_manager/server/lib/entities/ingest_pipeline/generate_history_processors.ts

```diff
@@ -132,6 +158,12 @@ export function generateHistoryProcessors(definition: EntityDefinition) {
         ignore_missing: true,
       },
     },
+    { script: { source: createIdentityFieldsPainlessScript(definition) } },
+    {
+      remove: {
+        field: 'entity.identityFields',
```

Comment:
Need to update how we generate the identity field based on this and swap the order.